packages/StackedBar/src/stacked-bar-tooltip.scss

```diff
@@ -0,0 +1,117 @@
+@use '~@igloo-ui/tokens/dist/base10/variables' as tokens;
+@use '~@igloo-ui/tokens/dist/fonts';
+
+:root {
+  /* Default */
+  --ids-stacked-bar-tooltip-font-family: #{tokens.$primary-font-family};
+  --ids-stacked-bar-tooltip-font-size: #{tokens.$font-size-7};
+  --ids-stacked-bar-tooltip-color: #{tokens.$grey-800};
+  --ids-stacked-bar-tooltip-background: #{tokens.$samoyed};
+  --ids-stacked-bar-tooltip-box-shadow: #{tokens.$shadow-12};
+  --ids-stacked-bar-tooltip-border-radius: #{tokens.$border-radius-sm};
+  --ids-stacked-bar-tooltip-padding: #{tokens.$space-3};
```

Comment:
`--ids-stacked-bar-tooltip-background`, `--ids-stacked-bar-tooltip-box-shadow`, `--ids-stacked-bar-tooltip-border-radius` and `--ids-stacked-bar-tooltip-padding` are unused